README.rst

```diff
@@ -235,28 +198,24 @@ build only the c++ library:
    cmake -DBUILD_BINDINGS=OFF ..
    make
 
-Installing rascal
-^^^^^^^^^^^^^^^^^
-To install the python library with c++ bindings:
+Helpers for Developers
```

Comment:
This should be in CONTRIBUTING, not README

Reply:
moved it into CONTRIBUTING, maybe double check if the changes in CONRTIBUTING make sense to you